integration-tests/src/test/java/com/example/IntegrationTest.java

```diff
@@ -727,6 +725,8 @@
 
   @Test public void providerCycle() {
     ignoreCodegenBackend();
+    // TODO this should work on reflection backend.
```

Comment:
No comment needed. All of these reflect ignores _should_ work eventually.

I should probably rename the methods to more clearly indicate that we ignore codegen because it causes compile-failures and we ignore reflect because it's a bug / missing feature.